src/network/networkd-link.c

```diff
@@ -755,7 +755,7 @@ void link_check_ready(Link *link) {
              !link->dhcp4_configured && !link->dhcp6_configured))
                 return;
 
-        if (link_ipv6_accept_ra_enabled(link) && !link->ndisc_configured)
+        if (link_ipv6_accept_ra_enabled(link) && !link->ndisc_configured && !link->network->bridge)
```

Comment:
can you elaborate why we only check this for IPv6 RA here? wouldn't it make sense to skip the whole section (including the dhcp/ipv4 stuff) if link->network->bridge is set?

Reply:
yes since we disable both .